config/default.conf

```diff
@@ -48,7 +48,7 @@ server.port:integer=3000
 
 #session
 # should be unique for each environment
-session.secret:string=mustbechanged
+# session.secret:string=mustbechanged
```

Comment:
I've been taking a look at what the projects have adopted and the most accepted strategy is the use of a signature key rotation mechanism to make the key exchange process smoother. 
Perhaps as a future feature this will be interesting for us:
https://www.npmjs.com/package/keygrip